SVGBezierPath.mm

```diff
@@ -91,15 +91,13 @@ - (SVGBezierPath *)pathBySettingSVGAttributes:(NSDictionary *)attributes
     NSParameterAssert(attributes);
     
     SVGBezierPath *path = [self copy];
-    if (path) {
-        if (path->_svgAttributes.count > 0) {
-            path->_svgAttributes = [path->_svgAttributes mutableCopy];
-            if (attributes) {
-                [(NSMutableDictionary *)path->_svgAttributes setValuesForKeysWithDictionary:attributes];
-            }
-        } else {
-            path->_svgAttributes = [attributes copy];
+    if (path->_svgAttributes.count > 0) {
+        path->_svgAttributes = [path->_svgAttributes mutableCopy];
+        if (attributes) {
```

Comment:
Remove this because of the assert at the beginning.

Reply:
done